tree-view-sample-plugin/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "tree-view-sample-plugin",
+  "publisher": "theia",
+  "keywords": [
+    "theia-plugin"
+  ],
+  "version": "0.0.1",
+  "license": "[object Object]",
```

Comment:
What is mean `[object Object]`?

Reply:
Thanks for this. It's a bug of our plugin generator.
I will change it on "EPL-2.0".

Reply:
AFAIK this issue has been fixed in our generator (so maybe an outdate version of the generator)